tortoisebeacon/weakcoin/weak_coin.go

```diff
@@ -4,290 +4,303 @@ import (
 	"bytes"
 	"context"
 	"encoding/binary"
+	"encoding/hex"
 	"fmt"
-	"math/big"
-	"strings"
 	"sync"
 
-	"github.com/ALTree/bigfloat"
 	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/common/util"
 	"github.com/spacemeshos/go-spacemesh/log"
-	"github.com/spacemeshos/go-spacemesh/p2p/service"
+	"github.com/spacemeshos/go-spacemesh/signing"
 )
 
 const (
 	// Prefix defines weak coin proposal prefix.
 	proposalPrefix = "WeakCoin"
 	// GossipProtocol is weak coin Gossip protocol name.
 	GossipProtocol = "WeakCoinGossip"
+
+	defaultThreshold = "0x8000000000000000000000000000000000000000000000000000000000000000"
 )
 
-type epochRoundPair struct {
-	EpochID types.EpochID
-	Round   types.RoundID
+// DefaultConfig ...
+func DefaultConfig() Config {
+	return Config{
+		Threshold:           util.FromHex(defaultThreshold),
+		VRFPrefix:           proposalPrefix,
+		NextRoundBufferSize: 10000, // ~1mb given the size of Message is ~100b
+		MaxRound:            100,
```

Comment:
Why is it 100?

Reply:
it is just random number. it doesn't have to be a valid number for spacemesh protocol. valid number must be configured during initalization

Reply:
Maybe 300 then? https://github.com/dshulyak/go-spacemesh/blob/finalize-weakcoin/tortoisebeacon/config.go#L23